websockets/conversation.go

```diff
@@ -146,6 +163,13 @@ func (c *Conversation) handleEditUpdate(msg protocol.Message, sender *Client) er
 		return err
 	}
 
+	newCheckpoint := &Checkpoint{
+		activeUsers: make(map[int64]protocol.Caret),
+		senderCaret: sender.caret,
+		delta:       *update.Delta,
+		syncsLeft:   make(map[int64]bool),
+	}
+
 	// Update the sender's caret
```

Comment:
I mentioned this in the PR for the WebSocket protocol doc: I think that applying the delta to the sender's cursor needs to be done *after* the shifting of all the other users' cursors. I did the steps in the wrong order when I wrote it.